tests/foreman/cli/test_leapp_client.py

```diff
@@ -217,6 +217,8 @@ def precondition_check_upgrade_and_install_leapp_tool(custom_leapp_host):
         )
 
 
+@pytest.mark.e2e
+@pytest.mark.skip_if_open('SAT-24023')
```

Comment:
this issue doesn't exist for 6.15.z, so can we remove this skip if open?

Reply:
Updated in https://github.com/SatelliteQE/robottelo/pull/15137.

Reply:
Updated in this PR.